.github/workflows/push_to_docker_hub.yml

```diff
@@ -0,0 +1,143 @@
+# This workflow replaces $SAGE_ROOT/.gitlab-ci.yml which has been
+# used formerly to make Docker images available on DockerHub.
+# The new workflow uses the same $SAGE_ROOT/docker/Dockerfile and
+# is based on docker/build-push-action@v4.
+#
+# A replacement has become neccessary because the GitLab runners
+# stopped to create new images since release 9.7.
+#
+# Concerning the images placed in the sagemath-dev repository you
+# should note the following difference to the releases build before
+# 9.8: They are no longer build on target sagemath-dev since this
+# causes a "No space left on device" error. Instead they are build
+# on target make-build, now. The main difference is the missing
+# documentation and that it enters in a bash shell.
+#
+# Sebastian Oehms, August 2023
+
```

Comment:
```suggestion
```

Thanks for the information. I believe that this is not terribly relevant anymore in a few months for anyone so maybe just drop it.

Could we delete the gitlab setup in this PR as well? It's not working anymore anyway.

Reply:
> Thanks for the information. I believe that this is not terribly relevant anymore in a few months for anyone so maybe just drop it.

No problem!

> Could we delete the gitlab setup in this PR as well? It's not working anymore anyway.

Sure! What should be dropped explicitly?